src/Platform.Handlers/src/Xamarin.Platform.Handlers/Platform/Standard/ButtonExtensions.cs

```diff
@@ -16,5 +11,40 @@ public static void UpdateText(this object nothing, IButton button)
 		{
 
 		}
+
+		public static void UpdateFont(this object nothing, IButton button)
```

Comment:
Are we actively using all of this on this PR? If not can we remove this for now?